proposals/cri-o.adoc

```diff
@@ -0,0 +1,221 @@
+== CRI-O
+
+*Name of project*: cri-o
+
+*Description*:
+
+CRI-O is a stable container runtime specifically tailored for Kubernetes.
+A founding principal was not "reinventing the wheel" and so the project uses shared components, refined approaches tested in production, and existing battle-tested code.
+Further, it is tuned for performance, stability, compatibility, and adherence to standards, particularly the Kubernetes Conformance tests.
+In short CRI-O is a building block of a Kubernetes cluster, to facilitate the life cycle of containers as required by the Kubernetes Container-Runtime-Interface (CRI).
+
+CRI-O was born in kubernetes-incubator and grew to production stable there.
+Some have claimed since being a kubernetes project that it is already "a CNCF project", but it is often left as an after thought in brand, awareness and support as a top-level project.
+This proposal is effective to promote from the kubernetes incubator a project that has fully matured.
+
+*Sponsor / Advisor from TOC*: Jonathan Boulle <jon@nstack.com>
+
+*Unique Identifier*: cri-o
+
+*License*: ALv2
+
+*Maturity Level*: Incubating
+
+*Source control repositories*: https://github.com/kubernetes-sigs/cri-o
+
+*Initial Committers*:
+
+- mrunalp
+- runcom
+- cyphar
+- mikebrow
+- sameo
+- rhatdan
+- nalind
+- giuseppe
+
+*Infrastructure requirements*: CI, CNCF Cluster, TravisCI
+
+*Issue tracker*: https://github.com/kubernetes-sigs/cri-o/issues
+
+*Mailing lists*
+
+* #cri-o on Freenode IRC
+* https://kubernetes.slack.com/messages/crio
+
+*Website*: http://cri-o.io
+
+*Release methodology and mechanics*
+
+Major releases are matching to Kubernetes major releases.
+Minor releases are as needed.
+
+*Social media accounts*:
+
+* Blog: https://medium.com/cri-o/
+
+*Existing sponsorship*:
+
+* Red Hat (lead)
+* Intel
+* SuSE
+
+*Adopters*:
+
+* Red Hat - Default container runtime for their OpenShift product
+* Intel
+* kata-containers - primary CRI runtime worked with
+* SuSE - Default for kubic and on track for their CaaSP offering (https://kubic.opensuse.org/blog/2018-09-17-crio-default/)
```

Comment:
`SUSE`, and [this is probably a better link for CaaSP where it's a tech preview at the moment](https://www.suse.com/releasenotes/x86_64/SUSE-CAASP/3.0/#Intro.New). I would list openSUSE separately, but that's just me.